web/src/user_profile.js

```diff
@@ -220,6 +220,8 @@ function render_user_stream_list(streams, user) {
         },
         $simplebar_container: $("#user-profile-modal .modal__body"),
     });
+    $(".ind-tab").attr("tabindex", "-1");
+    $(".modal__body .simplebar-content-wrapper").attr("tabindex", "-1");
```

Comment:
We should be setting properties like this in the HTML template where possible.

Also, does this break navigating between tabs with the arrow keys?

Reply:
@timabbott  I tried setting tabindex to -1 for `ind-tab` but there's no such element in HTML template. `components.ts` adds this class here
![image](https://github.com/zulip/zulip/assets/91622060/48884400-330c-4270-89c9-66fd68e0a7ae)
And I checked it doesn't break navigating between the tabs
 
![Animation6](https://github.com/zulip/zulip/assets/91622060/fdf3d852-8e9b-4a97-a8ab-ec6aba4919f3)